staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -236,8 +236,9 @@ const (
 // executing at each point where there is a change in that quantity,
 // because the metrics --- and only the metrics --- track that
 // quantity per FlowSchema.
-func (qs *queueSet) StartRequest(ctx context.Context, hashValue uint64, flowDistinguisher, fsName string, descr1, descr2 interface{}, queueNoteFn fq.QueueNoteFn) (fq.Request, bool) {
-	// all request(s) have a width of 1, in keeping with the current behavior
+func (qs *queueSet) StartRequest(ctx context.Context, _ float64, hashValue uint64, flowDistinguisher, fsName string, descr1, descr2 interface{}, queueNoteFn fq.QueueNoteFn) (fq.Request, bool) {
+	// TODO: "width" is overwritten to 1 for now, in keeping with the current behavior,
+	//  wire in the input parameter once we have actual "width" handling in place.
```

Comment:
Why not eliminate the overwrite in this PR, since this PR adds plumbing that always provides a value of 1?
Eliminating the overwrite would make it possible to write tests of non-unit width.

Reply:
+1

Reply:
I eliminated it